plebnet/settings/configuration/plebnet_setup.cfg

```diff
@@ -51,9 +51,13 @@ password = plebnet
 btc_global = SET_BTC_WALLET_ADDRESS_HERE
 mb_global = SET_MB_WALLET_ADDRESS_HERE
 
+[strategy]
+name = last_day_sell
+no_vps = 1
```

Comment:
I would consider renaming `no_vps` to `vps_count`, so it is clear without explanation what it means.